webpack/scenes/ContentViews/Details/Versions/__tests__/contentViewVersions.test.js

```diff
@@ -152,6 +152,7 @@ test('Can show additional content and link to list page', async () => {
       .toHaveAttribute('href', '/content_views/5#/versions/11/files/');
     expect(getByText('1 Deb packages').closest('a'))
       .toHaveAttribute('href', '/versions/11/debPackages');
+    expect(getByText(/80 Python packages\s*2 OSTree refs/)).not.toBeFalsy();
```

Comment:
I'm using a regex here because finding the exact match for white space in between  the python packages and the ostree refs was not  working.